src/zenml/zen_server/routers/artifact_version_endpoints.py

```diff
@@ -17,9 +17,20 @@
 from uuid import UUID
 
 from fastapi import APIRouter, Depends, Security
+from fastapi.responses import FileResponse
 
-from zenml.artifacts.utils import load_artifact_visualization
-from zenml.constants import API, ARTIFACT_VERSIONS, BATCH, VERSION_1, VISUALIZE
+from zenml.artifacts.utils import (
+    create_artifact_archive,
+    load_artifact_visualization,
+)
+from zenml.constants import (
+    API,
+    ARTIFACT_VERSIONS,
+    BATCH,
+    DATA,
```

Comment:
Maybe "download" instead of "data"?

Reply:
@AlexejPenner would disagree, because the naming of the endpoint should be a resource and not an action. Personally I don't really care so much, all of them work I think